x-pack/legacy/plugins/apm/server/lib/index_pattern/get_dynamic_index_pattern.ts

```diff
@@ -80,7 +80,11 @@ function getPatternIndices(
 ) {
   const indexNames = processorEvent
     ? [processorEvent]
-    : ['transaction' as const, 'metric' as const, 'error' as const];
+    : [
+        ProcessorEvent.transaction as const,
+        ProcessorEvent.metric as const,
+        ProcessorEvent.error as const
+      ];
```

Comment:
`as const` is no longer needed here as you are using an enum instead of a literal:
```ts
const indexNames = processorEvent
    ? [processorEvent]
    : [ProcessorEvent.transaction, ProcessorEvent.metric, ProcessorEvent.error];
```

Reply:
Done.